### PR TITLE
Update UG

### PR DESCRIPTION
**Pull request by hengyongming**

### CODE REVIEW

**Review by CSmortal — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -130,6 +130,8 @@ or suggestions [here](https://github.com/AY2021S2-CS2103T-W14-4/tp/issues).
 
 * Adding a task in the `home` tab will display the full list of tasks.
 
+* A task is considered duplicated if it's name, description and date is the same.
```
Comment (noelmathewisaac) [inline docs/UserGuide.md]: are the same

File: docs/UserGuide.md [hunk 2]
```diff
@@ -196,8 +198,7 @@ Format: `edit INDEX [n/NAME] [desc/DESCRIPTION] [date/DATE] [s/STATUS] [t/TAG]
 * You can remove all the task’s tags by typing `t/` without specifying any tags after it.
 * There are 3 Task statuses: `expired`, `uncompleted` and `completed`. However, you can cannot directly modify a 
   task's status to `expired` or from `uncompleted`/`completed` to `expired`.
-* If the edited task exist in the tasklist, an error will be thrown to the user. A task is considered duplicated only if 
-  it's name, description and date is the same.
+* If the edited task exist in the tasklist, an error will be thrown to the user.
```
Comment (noelmathewisaac) [inline docs/UserGuide.md:201]: After editing a task, if it is a duplicate of another task currently in the list, an error will be thrown to the user.

Comment (noelmathewisaac) [inline docs/UserGuide.md]: After editing a task, if it is a duplicate of another task currently in the list, an error will be thrown to the user.

**Review by CSmortal — APPROVED** [review]
LGTM

**Review by noelmathewisaac — APPROVED** [review]
LGTM

**Pull request merged**